test/AElf.Types.Tests/BinaryMerkleTestTest.cs

```diff
@@ -0,0 +1,301 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace AElf.Types.Tests
+{
+    public class BinaryMerkleTestTest
```

Comment:
Would it be better if we tested the number of nodes (count) is correct after generating a Merkle Tree. Like, from 4 leafs, 7 nodes; from 5 leafs, 13 nodes?